react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -109,6 +110,30 @@ const Map: React.FC<MapProps> = ({
         setSpecObj(jsonConverter.convert(deckglSpec));
     }, [deckglSpec, resources]);
 
+    const [discreteData, setDiscreteData] = React.useState([]);
+    const [dataPresent, setDataPresent] = React.useState(false);
+
+    React.useEffect(() => {
+        if (deckglSpec) {
+            //eslint-disable-next-line
+            const logData = (deckglSpec["layers"] as any[])[2].logData;
```

Comment:
use it as so there will not be multiple type casting 
const wells_layer = (deckglSpec["layers"] as any[])[2]);
const log_data = wells_layer.logData;
const log_data = wells_layer.logName;

also avoid using explicit indexing, use layer id to find the layer. From python dash side the layer ordering maybe different.